conf/messages.cy

```diff
@@ -1240,3 +1240,88 @@ rentalsAndRentARoomIncomeStart.l3 = *Missing Welsh*
+ukRentARoom.expensesIntro.p5.agent = Treuliau unigol, lle rydych yn nodi swm ar gyfer pob math o draul ar wahân.
+
+claimExpensesOrRRR.title = Hawlio treuliau neu Ryddhad Rhentu Ystafell
+claimExpensesOrRRR.heading.agent = Hawlio treuliau neu Ryddhad Rhentu Ystafell
+claimExpensesOrRRR.heading.individual = Hawlio treuliau neu Ryddhad Rhentu Ystafell
+claimExpensesOrRRR.checkYourAnswersLabel.amount.individual = Faint yr hoffech ei hawlio?
+claimExpensesOrRRR.checkYourAnswersLabel.amount.agent = Faint hoffai’ch cleient ei hawlio?
+claimExpensesOrRRR.checkYourAnswersLabel.type = Hawlio treuliau neu Ryddhad Rhentu Ystafell
+claimExpensesOrRRR.no = Treuliau
+claimExpensesOrRRR.yes = Rhyddhad Rhentu Ystafell
+claimExpensesOrRRR.legend.individual = Pa un yr hoffech ei hawlio?
+claimExpensesOrRRR.legend.agent = Pa un hoffai’ch cleient ei hawlio?
+claimExpensesOrRRR.error.required.individual = Dewiswch os hoffech hawlio treuliau neu Ryddhad Rhentu Ystafell
+claimExpensesOrRRR.error.required.agent = Dewiswch os hoffai’ch cleient hawlio treuliau neu Ryddhad Rhentu Ystafell
+claimExpensesOrRRR.amount.error.required.individual = Nodwch swm y Rhyddhad Rhentu Ystafell yr hoffech ei hawlio
+claimExpensesOrRRR.amount.error.required.agent = Nodwch swm y Rhyddhad Rhentu Ystafell yr hoffai’ch cleient ei hawlio
+claimExpensesOrRRR.amount.error.nonNumeric.individual = Gall cyfanswm y Rhyddhad Rhentu Ystafell a hawliwyd gynnwys punnoedd a cheiniogau yn unig, er enghraifft ¬£600 neu ¬£600.20
+claimExpensesOrRRR.amount.error.nonNumeric.agent = Gall cyfanswm y Rhyddhad Rhentu Ystafell a hawliwyd gynnwys punnoedd a cheiniogau yn unig, er enghraifft ¬£600 neu ¬£600.20
+claimExpensesOrRRR.amount.error.twoDecimalPlaces.individual = Gall cyfanswm y Rhyddhad Rhentu Ystafell a hawliwyd gynnwys punnoedd a cheiniogau yn unig, er enghraifft ¬£600 neu ¬£600.20
+claimExpensesOrRRR.amount.error.twoDecimalPlaces.agent = Gall cyfanswm y Rhyddhad Rhentu Ystafell a hawliwyd gynnwys punnoedd a cheiniogau yn unig, er enghraifft ¬£600 neu ¬£600.20
+claimExpensesOrRRR.amount.error.outOfRange = Nodwch swm rhwng ¬£{0} a ¬£{1}
+claimExpensesOrRRR.change.hidden = Hawlio treuliau neu Ryddhad Rhentu Ystafell
+claimExpensesOrRRR.rentARoom = Rhyddhad Rhentu Ystafell
+claimExpensesOrRRR.expenses = Treuliau
+claimExpensesOrRRR.radio.question.label.individual = Faint yr hoffech ei hawlio?
+claimExpensesOrRRR.radio.question.label.agent = Faint hoffai’ch cleient ei hawlio?
+claimExpensesOrRRR.para1.jointlyLet.individual = Gallwch hawlio hyd at ¬£3,750 y flwyddyn yn rhydd o dreth drwy rentu ystafell yn eich cartref. Os byddwch yn hawlio’r rhyddhad hwn, ni fyddwch yn gallu hawlio treuliau ar gyfer yr un ystafell.
+claimExpensesOrRRR.para1.notJointlyLet.individual = Gallwch hawlio hyd at ¬£7,500 y flwyddyn yn rhydd o dreth drwy rentu ystafell yn eich cartref. Os byddwch yn hawlio’r rhyddhad hwn, ni fyddwch yn gallu hawlio treuliau ar gyfer yr un ystafell.
```

Comment:
Content changed in English, pls compare English text and the one we requested. Pls test all screens

Refer this commit: f1c3d32324b6a2f1b834b8b12047533aacfd2c48

Reply:
I've removed the unused messages, and parameterised the first two ones on the Welsh translation. Regarding testing all screens, I believe it is QA's responsibility, however, sure I'll do a short check to make sure the Welsh translation is working.

Reply:
thanks @modivilioglu for addressing the issue, when I mean all screens, the files you have changed part of this PR